data/human/hails.txt

```diff
@@ -5824,6 +5827,281 @@ phrase "friendly deep"
 		"piracy"
 		"social regression"
 		"corruption"
+		"hostility and war"
+		"instability"
+		"anarchism"
+		"hostility"
+		"base values"
+	word
+		"."
+
+phrase "friendly deep"
+	word
+		"We"
+		"Our ships"
+		"Our security forces"
+		"Our fleets"
+		"Our forces"
+	word
+		" "
+	word
+		"" 3
+		"are called to "
+		"are diligent to "
+		"must constantly "
+		"diligently "
+		"always "
+		"must "
+		"shall "
+	word
+		"safeguard"
+		"defend"
+		"protect"
+		"guard"
+		"shield"
+		"advance"
+		"work in advancing"
+	word
+		" "
+	word
+		"" 3
+		"the Deep's "
+		"those who bring "
+		"the ones who serve "
+		"agents of "
+		"our shared "
+		"our values of "
+	word
+		""
+		"valued "
+		"invaluable "
+		"hard-fought "
+		"hard-gained "
```

Comment:
This can lead to phrases like "our values of valued democracy" or "agents of invaluable democracy." I personally think these adjectives can be removed, but I don't know of a good solution.

Reply:
I'll remove 'agents of' instead, it should be better.

Reply:
"our values of" should also be removed if this is the solution
```suggestion
	word
		"" 3
		"the Deep's "
		"those who bring "
		"the ones who serve "
		"our shared "
	word
		""
		"valued "
		"invaluable "
		"hard-fought "
		"hard-gained "
```